src/pyhub/llm/utils/files.py

```
@@ -186,11 +187,33 @@ def encode_files(
+                    for img_data, img_mime_type in pdf_images:
+                        prefix = f"data:{img_mime_type};base64,"
+                        b64_string = b64encode(img_data).decode("utf-8")
+                        encoded_urls.append(f"{prefix}{b64_string}")
+
+                    continue
+                else:
+                    logger.warning(
+                        f"IO type not allowed: {content_type} for {file.name}. "
+                        f"Allowed types: {', '.join(allowed_mimetypes)}"
+                    )
+                    continue
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "continue", remove the "else" and de-indent the code inside it

(R1724)